deployment_specific/pb/ds_locales/en.js

```diff
@@ -171,63 +171,58 @@ const en = {
       'title': 'Watch Instructional Video'
     },
     'whatsapp': {
-      'title': 'Report via WhatsApp',
+      'title': 'Complete Training Exercise via WhatsApp',
       'steps': [
-        'Send a message to +62858-4-BENCANA (+62 858-4-2362262) or <a target="_blank" href="https://bit.ly/BencanaBotWA">https://bit.ly/BencanaBotWA</a>',
+        'Send a message to +62858-4-BENCANA (+62 858-4-2362262) or https://bit.ly/BencanaBotWA',
```

Comment:
Make sure to have the link in a href with target blank

Reply:
Fixed the issue